### PR TITLE
Add docs for expanding PVC in AWS

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/giantswarm/issues/2550

### CODE REVIEW

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 1]
```diff
@@ -80,6 +80,14 @@ spec:
 
 Now we have an NGINX Pod which serves the contents of our EBS Volume.
 
+## Expanding Persistent Volumes Claims
+
+From Kubernetes v1.9 persistent volume claim can be expanded by simply editing the claim and requesting a larger size. It will trigger an update in the underlying persistent volume to make it possible (Kubernetes always use the existing one).
+
+In case the volume expanded contains a file system, the resizing is only performed when a new Pod is started using the persistent volume claim in `ReadWrite` mode. In other words, if a volume being expanded is used in a pod or deployment, you will need to delete and recreate the pod for file system resizing to take place. File system resizing is only supported for XFS, Ext3 and Ext4.
+
+__Warning__ Expanding EBS volumes is a time consuming operation. Also, there is a per-volume quota of one modification every 6 hours.
```
Comment (marians) [inline content/guides/using-persistent-volumes-on-aws/index.md]: Please add a colon `:` behind `__Warning__`.

**Comment by marians** [comment]
Cool!

I'd appreciate a sentence explaining what "expansion" means, pretty much at the beginning of this section. I have an idea what it means, but I don't wanna be guessing.

**Comment by puja108** [comment]
@marians it says "requesting a larger size" in the first sentence

@pipo02mix there's quite some language things to correct, mind if I do the edits miyself instead of commenting?

**Comment by pipo02mix** [comment]
@puja108 yes please go ahead
thx

**Comment by pipo02mix** [comment]
Thanks for the improvements @puja

**Pull request merged**